driver-core/src/main/java/com/datastax/driver/core/ReplicationStategy.java

```diff
@@ -55,6 +56,7 @@ static ReplicationStrategy create(Map<String, String> replicationOptions) {
             }
         } catch (NumberFormatException e) {
             // Cassandra wouldn't let that pass in the first place so this really should never happen
+            logger.error("Failed to parse replication options: " + replicationOptions, e);
```

Comment:
We did not reproduce a NumberFormatException. All that we did was add a log where an exception was silently being swallowed. Sylvain added that catch block as part of rev 1497aedfb4d29277489bc285b3642a4387b28f3b. I am not sure about the implications of removing that block. Let me know if you are OK leaving it there as part of this PR. Feel free to remove the catch block in another commit.